test/benchmark.js

```diff
@@ -3,9 +3,7 @@ const { tmpdir } = require('os');
 const { mkdirSync, writeFileSync } = require('fs');
```

Comment:
I'm not even sure what this file is. Perhaps we should delete it if it's not being used by CI.